Gitpod starts with ruby 3.2.2, but bulletrain requires 3.3.0

Steps to reproduce:
1. Click on gitpod icon on repo homepage
2. Select default gitpod settings
3. See errors in terminal

Updating _ruby_version=ruby-3.3.0 in gitpod.Dockerfile seems to help, but I gitpod gets stuck during validation here:

```
=========================================================================================================================
[2024-04-09 19:31:04] ERROR 19:31:04: Could not load vsce-sign module Cannot find module '@vscode/vsce-sign'
[2024-04-09 19:31:04] ERROR Require stack:
[2024-04-09 19:31:04] ERROR - /ide/out/server-main.js
[2024-04-09 19:31:04] ERROR 19:31:04: Could not load vsce-sign module Cannot find module '@vscode/vsce-sign'
[2024-04-09 19:31:04] ERROR Require stack:
[2024-04-09 19:31:04] ERROR - /ide/out/server-main.js

```

--- a/gitpod.Dockerfile
+++ b/gitpod.Dockerfile
@@ -1,7 +1,7 @@
 FROM gitpod/workspace-postgres
 USER gitpod
 
-RUN _ruby_version=ruby-3.2.2 \
+RUN _ruby_version=ruby-3.3.0 \
     && printf "rvm_gems_path=/home/gitpod/.rvm\n" > ~/.rvmrc \
     && bash -lc "rvm reinstall ruby-${_ruby_version} && rvm use ruby-${_ruby_version} --default && gem install rails" \
     && printf "rvm_gems_path=/workspace/.rvm" > ~/.rvmrc \

--- a/gitpod.Dockerfile
+++ b/gitpod.Dockerfile
@@ -1,7 +1,7 @@
 FROM gitpod/workspace-postgres
 USER gitpod
 
-RUN _ruby_version=ruby-3.3.0 \
+RUN _ruby_version=ruby-3.3.1 \
     && printf "rvm_gems_path=/home/gitpod/.rvm\n" > ~/.rvmrc \
     && bash -lc "rvm reinstall ruby-${_ruby_version} && rvm use ruby-${_ruby_version} --default && gem install rails" \
     && printf "rvm_gems_path=/workspace/.rvm" > ~/.rvmrc \

